### PR TITLE
feat(cms): Adding a projects-wide event service

### PR DESCRIPTION
**Pull request by Varun-Sethu**
PR just adds a projects wide event service

### CODE REVIEW

File: services/events-service/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+# `-tags timetzdata` embeds zone info from the "time/tzdata" package.
+RUN CGO_ENABLED=0 go install -ldflags '-extldflags "-static"' -tags timetzdata
+
+FROM scratch
+# the test program:
+COPY --from=app-builder /go/bin/events-service /events-service
+# the tls certificates:
+# NB: this pulls directly from the upstream image, which already has ca-certificates:
+COPY --from=alpine:latest /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
+ENTRYPOINT ["/events-service"]
```
Comment (lhjt) [inline services/events-service/Dockerfile:15]: Can you add a new line to the end of this file? Otherwise all good 😎

**Review by lhjt — CHANGES_REQUESTED** [review]

File: services/events-service/Dockerfile (hunk 1 above)
Comment (Varun-Sethu) [reply services/events-service/Dockerfile:15]: Bro you're too fast but yup 😎

Comment (Varun-Sethu) [reply services/events-service/Dockerfile]: Alright updated 😎

**Review by lhjt — APPROVED** [review]
Incredible

**Pull request merged**